src/main/kotlin/eu/europa/ec/eudi/openid4vci/Issuance.kt

```diff
@@ -165,7 +134,7 @@ interface RequestIssuance {
      * @param popSigner Signer component of the proof to be sent. Although this is an optional
      * parameter, only required in case the present authorization state is [AuthorizedRequest.ProofRequired],
      * caller is advised to provide it, in order to allow the method to automatically retry
-     * in case of [SubmissionOutcome.InvalidProof]
+     * in case of [CredentialIssuanceError.InvalidProof]
      *
      * @return the possibly updated [AuthorizedRequest] (if updated it will contain a fresh c_nonce) and
      * the [SubmissionOutcome]
```

Comment:
Shall this be renamed to AuthorizedRequest.requestBatch()? Since the default behavior is to update the state internally I guess there is no need to state it on the method name